lib/specs/canary.js

```diff
@@ -35,6 +35,15 @@ let rules = {
         details: oneLineTrim`Please change <code>"ghost-api"</code> in your <code>package.json</code> to higher version. E.g. <code>{"engines": {"ghost-api": "v4"}}</code>.<br>
         If <code>"ghost-api"</code> property is left at "v2", it will stop working with next major version upgrade and default to v5.<br>
         Check the <a href="${docsBaseUrl}packagejson/" target=_blank><code>package.json</code> documentation</a> for further information.`
+    },
+    'GS001-DEPR-LABS-MEMBERS': {
+        level: 'warning',
+        rule: 'The <code>{{@labs.members}}</code> helper should not be used and removed from the theme.',
```

Comment:
@matthanley could you please give this a quick copy review .The context is: we deprecated {{@labs.members}} with Ghost 4.0 and would like to give a nice warning for people to cleanup their themes before we completely remove the flag in next Ghost version. The reference I used to make up the message here was this other similar warnings:
```
'GS001-DEPR-IUA': {
    level: 'error',
    rule: 'Replace <code>{{img_url author.*}}</code> with <code>{{img_url primary_author.*}}</code> or <code>.{img_url author.[#].*}}</code>',
    details: oneLineTrim`The usage of <code>{{img_url author.*}}</code> is deprecated and should be replaced with either <code>{{img_url primary_author.*}}</code>
    or <code>{{img_url author.[#].*}}</code>.<br>
    Find more information about the <code>{{authors}}</code> helper <a href="${docsBaseUrl}helpers/authors/" target=_blank>here</a>.`,
    regex: /{{\s*?img_url\s*?(author.).*}}/g,
    helper: '{{img_url author.*}}'
},
'GS001-DEPR-BLOG': {
    level: 'error',
    rule: 'The <code>{{@blog}}</code> helper should be replaced with <code>{{@site}}</code>',
    details: oneLineTrim`With the introduction of the Content API <code>{{@blog}}</code> became deprecated in favour of <code>{{@site}}</code>.<br>
    The <code>{{@blog}}</code> helper will be removed in next version of Ghost and should not be used.
    Find more information about the <code>@site</code> property <a href="${docsBaseUrl}helpers/site/" target=_blank>here</a>.`,
    regex: /{{\s*?@blog\.[a-zA-Z0-9_]+\s*?}}/g,
    helper: '{{@blog}}'
}
```